classes/blocks/class-columns.php

```diff
@@ -79,7 +79,7 @@ public function __construct() {
 		add_shortcode( 'shortcake_columns', [ $this, 'add_block_shortcode' ] );
 
 		register_block_type(
-			'planet4-blocks/columns',
+			self::BLOCK_NAMESPACE_PREFIX . '/' . self::BLOCK_NAME,
```

Comment:
We can put this into the base block class in a function since it's the same for all blocks.

Reply:
In fact we can probably move the entire `register_block_type` call into the base class.

Reply:
Though that would increase this PR's size a lot, just replacing all concatenations with a function is maybe better for now.

Reply:
Sounds good.